package-lock.json

```diff
@@ -1,24 +1,1639 @@
 {
   "name": "clickhouse",
   "version": "2.5.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
What is point for package version changes?

Reply:
Hi TimonKK,

The npm version that I've installed (8) is causing the change.
No problems are expected because the lockfileVersion 2 is backwards compatible with version 1.
https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion

Thanks in advance